### DISCUSSION TITLE
Link Length Limit on Import

### DISCUSSION THREAD

**Discussion by motusRP** [inventree/InvenTree#7078] [Q&A]
I'm trying to import a list of Supplier Parts that has links to the supplier's websites.  Some of the links are over 200 characters in length (so far, i've seen 201, 208, 215).  Is there a way to increase the limit for these links?

![image](https://github.com/inventree/InvenTree/assets/54725635/23573c21-79d4-4df6-89ab-5612faece940)

[update] Interesting, I just tried importing the Manufacturer Parts using the same list of URL's and had no errors.  So the URL Length limit is different for Manufacturer Parts and Supplier Parts.

**Comment by matmair** [comment]
Default link length is 200 chars; we could expand that a bit, please raise a [feature request](https://github.com/inventree/InvenTree/issues/new?assignees=&labels=enhancement%2Ctriage%3Anot-checked&projects=&template=feature_request.yaml&title=%5BFR%5D+title) and ping me in the comments, I will take a look at fixing this for 0.15.0.

**Reply by matmair** [reply]
IMO Manufacturer parts should also be limited to 200 chars

**Reply by motusRP** [reply]
I submitted a feature request.  @matmair if both fields are limited to 200 characters, what solution would you suggest when a manufacturer or supplier's website requires a URL longer than 200 characters to identify their part?

**Reply by matmair** [reply]
Extending the field. I will take a look tonight. Should not be too difficult.
